pkg/httpserver/server.go

```diff
@@ -37,7 +37,10 @@ func New(url, certFile, keyFile string, handlers ...common.HTTPHandler) *Server
 	router := mux.NewRouter()
 	for _, handler := range handlers {
 		logger.Infof("Registering handler for [%s]", handler.Path())
-		router.HandleFunc(handler.Path(), handler.Handler()).Methods(handler.Method())
+
+		router.HandleFunc(handler.Path(), handler.Handler()).
```

Comment:
we should remember to add access control options (via Authorization header bearer token).